db/migrate/20230327200000_add_prohibit_temp_terms_to_controlled_vocabularies.rb

```diff
@@ -0,0 +1,7 @@
+class AddProhibitTempTermsToControlledVocabularies < ActiveRecord::Migration
+  def change
+    change_table(:controlled_vocabularies) do |t|
+      t.boolean :prohibit_temp_terms, default: false, null: false, index: true
```

Comment:
```suggestion
      t.boolean :prohibit_temp_terms, default: false, null: false
```
Maybe no need for an index on this field?